src/components/Alert/Alert.tsx

```diff
@@ -19,7 +19,7 @@ const Alert: React.FC<Props> = ({ open, message, onClose, isSuccess, actionsOver
   const { t } = useTranslation('common');
 
   return (
-    <Dialog open={open} onClose={onClose}>
+    <Dialog open={open} onClose={onClose} role="alert" aria-live="polite">
```

Comment:
A `role="alert"` shouldn't be combined with `aria-live="polite"`. It's one or the other.
Browsers will treat `role="alert`" as a dynamic change, causing it to be auto read by a screen reader.

Also see the note on https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions which also explains that

Reply:
I've removed `aria-live="polite"`